app-modules/alert/src/Observers/AlertObserver.php

```diff
@@ -56,12 +56,12 @@ public function created(Alert $alert): void
 
     public function saved(Alert $alert): void
     {
-        Cache::tags('alert-count')->flush();
+        Cache::tags('{{alert-count}}')->flush();
     }
 
     public function deleted(Alert $alert): void
     {
-        Cache::tags('alert-count')->flush();
+        Cache::tags('{{alert-count}}')->flush();
```

Comment:
This should only have one set of `{}` around this.

In the example you are copying from, it only has the double `{}` because it is within a string with a variable in it and String interpolation.